src/notification-toast/types.ts

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export type NotificationToast =
+    | "success"
+    | "warning"
+    | "error"
+    | "info"
+    | "successDark"
+    | "warningDark"
+    | "errorDark"
+    | "infoDark";
+
+export interface NotificationToastProps
+    extends React.HTMLAttributes<HTMLDivElement> {
+    type: NotificationToast;
+    title: string | undefined;
+    description: string | undefined;
+    icon: string | undefined;
```

Comment:
why is icon a string? i think it should use a component instead. 

Anyway we are deprecating our `Icon` component. Use the icon from `@lifesg/react-icons` instead